translations/ai_heroes.tex/pl.po

```diff
@@ -119,6 +143,19 @@ msgid ""
 "If an AI Hero is instructed to draw a card, they will draw and resolve \\textbf{another card} from the AI Deck.\n"
 "\n"
 msgstr ""
+"Bohaterowie AI używają dwóch Talii Kart podczas Walki:"
+"\\textbf{Talii AI} oraz \\textbf{Talii Magii}.\n"
+"Talia AI jest budowana z wykorzystaniem trzech rodzajów kart AI: Mocy \\includesvg[height=10px]{\\svgs/might.svg}, Magii \\includesvg[height=10px]{\\svgs/magic.svg} oraz Umiejętności \\includesvg[height=10px]{\\svgs/skill.svg}.\n"
+"Każdy Scenariusz Kampanii podaje liczbę każdego rodzaju Karty AI wchodzącej w skład Talii AI.\n"
+"Karty Mocy \\includesvg[height=10px]{\\svgs/might.svg} oraz Umiejętności \\includesvg[height=10px]{\\svgs/skill.svg} mają dwie wersje. Wybierz te karty \\textbf{losowo} w trakcie budowania Talii.\n"
+"Jeśli w Talii są karty Umiejętności, wyszukaj i wybierz je odkładając na bok."
+"Zbuduj \\textbf{Talię Magii AI}, oddzielając wskazane Zaklęcia od zwykłej Talii Magii.\n"
+"Po ich utworzeniu potasuj Talię AI i Talię Zaklęć podczas przygotowania gry.\\par\n"
```

Comment:
tak się zastanawiam, czego tyczy się "ich" w tym przypadku?

Reply:
Tyczy się liczby mnogiej - Talii, a więc "ich utworzenie"

Reply:
ich w sumie można usunąć, gdyż nie zmienia to sensu logiczno-składniowego zdania